src/@types/index.d.ts

```diff
@@ -1,7 +1,133 @@
+interface FetchResponse {
+  json(): Promise<any>;
+}
+
+//  Server response
+// ---------------------------------------------------------------------------
+
 interface Client {
   hasFeatureSupport: boolean;
 }
 
 interface Fastly {
   client: Client;
 }
+
+interface Host {
+  host: string;
+  lookup: string;
+}
+
+interface Settings {
+  initial_delay: number;
+  max_tasks: number;
+  report_errors: boolean;
+  sample_rate: number;
+}
+
+interface Server {
+  datacenter: string;
+}
+
+// Config/tasks
+// ---------------------------------------------------------------------------
+interface Task {
+  name: string;
+  req_header: string;
+  resource: string;
+  resp_header: string;
+  type: string;
+  weight: number;
+}
+
+interface Config {
+  apiKey: string;
+  session: string;
+  hosts: Host;
+  settings: Settings;
+  server: Server;
+  tasks: Task[];
+}
+
+// ClientInfo (Raw/cooked)
```

Comment:
These currently look identical, however `ClientInfo` should have the asn's as `number` etc.

Reply:
Oh, that's weird. I could swear I changed the types of those asn fields.